### PR TITLE
Rename io.github.mrquantumoff.mcmodpackmanager to dev.mrquantumoff.mcmodpackmanager

### PR DESCRIPTION
**Pull request by mrquantumoff**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by mrquantumoff** [comment]
bot, build

File: dev.mrquantumoff.mcmodpackmanager.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+modules:
+  - name: mcmodpackmanager
+    buildsystem: simple
+    build-commands:
+      - ar -vox minecraft-modpack-manager_*_amd64.deb
+      - tar -xpvf data.tar.gz
+      - install -Dm755 -t /app/bin usr/bin/minecraft-modpack-manager
+      - install -D dev.mrquantumoff.mcmodpackmanager.metainfo.xml /app/share/metainfo/dev.mrquantumoff.mcmodpackmanager.metainfo.xml
+      - sed -i 's/Icon=minecraft-modpack-manager/Icon=dev.mrquantumoff.mcmodpackmanager/g' usr/share/applications/minecraft-modpack-manager.desktop
```
Comment (hfiguiere) [inline dev.mrquantumoff.mcmodpackmanager.yml:24]: use `desktop-file-edit` instead of sed.

File: dev.mrquantumoff.mcmodpackmanager.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Flatpak Manifest
+app-id: dev.mrquantumoff.mcmodpackmanager
+runtime: org.gnome.Platform
+runtime-version: '42'
```
Comment (hfiguiere) [inline dev.mrquantumoff.mcmodpackmanager.yml:4]: 43 is the latest. 42 will be eol in April/May

File: dev.mrquantumoff.mcmodpackmanager.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      - ar -vox minecraft-modpack-manager_*_amd64.deb
+      - tar -xpvf data.tar.gz
+      - install -Dm755 -t /app/bin usr/bin/minecraft-modpack-manager
+      - install -D dev.mrquantumoff.mcmodpackmanager.metainfo.xml /app/share/metainfo/dev.mrquantumoff.mcmodpackmanager.metainfo.xml
+      - sed -i 's/Icon=minecraft-modpack-manager/Icon=dev.mrquantumoff.mcmodpackmanager/g' usr/share/applications/minecraft-modpack-manager.desktop
+      - install -D usr/share/applications/minecraft-modpack-manager.desktop  /app/share/applications/dev.mrquantumoff.mcmodpackmanager.desktop
+      - install -D usr/share/icons/hicolor/256x256/apps/com.mrquantumoff.mcmodpackmanager.png  /app/share/icons/hicolor/256x256/apps/dev.mrquantumoff.mcmodpackmanager.png 
+    sources:
+      - type: file
+        path: dev.mrquantumoff.mcmodpackmanager.metainfo.xml
```
Comment (hfiguiere) [inline dev.mrquantumoff.mcmodpackmanager.yml:29]: this file is still not managed upstreeam? I should be.

**Comment by hfiguiere** [comment]
is this really necessary to rename it? It's very disruptive for users.

File: dev.mrquantumoff.mcmodpackmanager.yml (hunk 3 above)
Comment (mrquantumoff) [reply dev.mrquantumoff.mcmodpackmanager.yml:29]: It will be resolved with 5.0.1 release

**Comment by mrquantumoff** [comment]
> is this really necessary to rename it? It's very disruptive for users.

I know, but using io.github instead of a domain I fully control is not as comfortable, because ~~if~~ when I want to verify my app on flathub in order for users to trust it a little more it'd be easier for me to use dev.mrquantumoff.

File: dev.mrquantumoff.mcmodpackmanager.yml (hunk 1 above)
Comment (mrquantumoff) [reply dev.mrquantumoff.mcmodpackmanager.yml]: done.

File: dev.mrquantumoff.mcmodpackmanager.yml (hunk 3 above)
Comment (mrquantumoff) [reply dev.mrquantumoff.mcmodpackmanager.yml:30]: it is resolved now.

**Comment by barthalion** [comment]
bot, build dev.mrquantumoff.mcmodpackmanager

**Comment by mrquantumoff** [comment]
image
strange, I don't get this error

**Comment by mrquantumoff** [comment]
Nevermind, there was a typo in the manifest

**Comment by mrquantumoff** [comment]
bot, build dev.mrquantumoff.mcmodpackmanager

File: dev.mrquantumoff.mcmodpackmanager.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.mrquantumoff.mcmodpackmanager</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>BSD-2-Clause</project_license>
+  <name>Minecraft Modpack Manager</name>
+  <summary>Manage your modpacks with ease!</summary>
+  <icon type="remote" width="256" height="256">
```
Comment (hfiguiere) [inline dev.mrquantumoff.mcmodpackmanager.metainfo.xml:8]: please don't use remote icons.

Comment (mrquantumoff) [reply dev.mrquantumoff.mcmodpackmanager.metainfo.xml:8]: is using stock icons ok?

Comment (mrquantumoff) [reply dev.mrquantumoff.mcmodpackmanager.metainfo.xml:8]: So, which type of icons should I use? If I have to use "cached" or "local" will it be correctly displayed with GNOME Software/Discover?

Comment (hfiguiere) [reply dev.mrquantumoff.mcmodpackmanager.metainfo.xml:8]: just remove the icon entry.

Comment (mrquantumoff) [reply dev.mrquantumoff.mcmodpackmanager.metainfo.xml:8]: Ok

**Comment by mrquantumoff** [comment]
Queued test build for dev.mrquantumoff.mcmodpackmanager.

**Comment by mrquantumoff** [comment]
bot, build dev.mrquantumoff.mcmodpackmanager

**Comment by mrquantumoff** [comment]
Queued test build for dev.mrquantumoff.mcmodpackmanager.

**Comment by mrquantumoff** [comment]
bot, build dev.mrquantumoff.mcmodpackmanager

**Comment by mrquantumoff** [comment]
bot, build dev.mrquantumoff.mcmodpackmanager

**Comment by mrquantumoff** [comment]
bot, build dev.mrquantumoff.mcmodpackmanager

File: dev.mrquantumoff.mcmodpackmanager.yml (hunk 2 above)
Comment (mrquantumoff) [reply dev.mrquantumoff.mcmodpackmanager.yml]: I rewrote the app in Flutter/Dart, so now it's resolved (v6.0.0 uses freedesktop runtime)

**Comment by mrquantumoff** [comment]
bot, build dev.mrquantumoff.mcmodpackmanager

**Comment by mrquantumoff** [comment]
bot, build dev.mrquantumoff.mcmodpackmanager

**Comment by barthalion** [comment]
/merge

Don't forget to EOL the old ref and open an issue to archive the deprecated repo.

**Pull request closed without merging**